### ISSUE TITLE
More maintainers

### ISSUE THREAD

**Issue by Uriopass** [Uriopass/meshopt2#1]
I am open to add more maintainers to this project (github & crates.io rights) that want to have the latest bindings so that it doesn't depend on a single person. 

Ping @BeastLe9enD, @oisyn and @gents83 as you recently contributed new bindings/changes to meshopt

**Comment by BeastLe9enD** [comment]
Considering the fact that the crate on Github was last updated over a year ago (and November 2019 on crates.io), I think the is the best way to go. I would like to help as a maintainer!

**Comment by oisyn** [comment]
Same here!

**Comment by oisyn** [comment]
Have you tried reaching out to @gwihlidal to see if he was maybe interested in handing over the crates.io rights at least?

**Comment by BeastLe9enD** [comment]
good point, maybe we could try asking him on twitter (I think Github issues is not the best way to reach him)

**Comment by Uriopass** [comment]
You're right, I didn't think of that!

**Comment by Uriopass** [comment]
I'll send him a message on twitter

EDIT: Done

**Comment by Uriopass** [comment]
In the meantime, I have added you both to meshopt2 on crates.io too :+1:

**Comment by BeastLe9enD** [comment]
@Uriopass Have you received an answer so far?

**Comment by Uriopass** [comment]
No answers :/

**Comment by gwihlidal** [comment]
Hey all,

Sorry I've been busy, but I'm definitely up for getting help maintaining this crate! I can add you all to this repo and also crates.io

I've been hoping to find some time again to resume crate dev, but getting your help will be awesome.

Thank you,
Graham

**Comment by Uriopass** [comment]
> I can add you all to this repo and also crates.io

That would be great! Thank you

**Comment by JMS55** [comment]
Hi. I'm using meshopt in Bevy for [meshlet generation](https://github.com/bevyengine/bevy/pull/10164). Currently, I'm using [my own fork](https://github.com/JMS55/meshopt-rs/commits/master) of meshopt that exposes some non-public fields and implements DecodePosition for an array of floats. In the future, I'm also going to need [Traverse Research's fork](https://github.com/Traverse-Research/meshopt-rs/tree/vertex-lock) that exposes simplifying meshes while locking edges (included in my branch atm, although I don't currently use it).

A prerequisite to being able to merge my code into Bevy is depending on a crates.io published crate, instead of a git fork. Would you be interested in merging my (small commits), and possibly Traverse Research's additional API, and then publishing a new crates.io release of meshopt2?

**Comment by Uriopass** [comment]
Absolutely open to that, these seem like good API changes!

It might be better to wait for @gwihlidal to give us maintaining control over the [original crate](https://github.com/gwihlidal/meshopt-rs) though, as it'd be a bit less confusing

**Comment by oisyn** [comment]
Keep in mind that our fork (I work at Traverse 😉) is dependent on https://github.com/zeux/meshoptimizer/pull/601 to be merged. We could of course opt to use the Traverse fork of the C++ library as basis in the mean time -- I would personally favor that as that would remove some patches in our Cargo.toml, but I'm a bit biased here 😄.

**Comment by gwihlidal** [comment]
I have added @Uriopass, @BeastLe9enD, @oisyn and @gents83 as maintainers to the meshopt repository, as well as crates.io - thank you very much for the help!
